code/modules/mob/living/silicon/robot/robot_defines.dm

```diff
@@ -130,6 +133,8 @@
 	var/hasExpanded = FALSE
 	var/obj/item/hat
 	var/hat_offset = -3
+	var/hat_offset_x = list()
+	var/hat_offset_y = list()
```

Comment:
If you're deadset on keeping the Y offset you should completely removing the existing `hat_offset` var since it's no longer needed

Reply:
I don't think we should do a list for every borg skin (there's at least 50), when the y-offset alone works fine for most of them. Also removing it would break downstreams badly.